torch/utils/hipify/cuda_to_hip_mappings.py

```diff
@@ -634,6 +634,7 @@
         ("cub/device/device_reduce.cuh", ("hipcub/hipcub.hpp", CONV_INCLUDE, API_BLAS)),
         ("cub/device/device_scan.cuh", ("hipcub/hipcub.hpp", CONV_INCLUDE, API_BLAS)),
         ("nvToolsExt.h", ("roctracer/roctx.h", CONV_INCLUDE, API_ROCTX)),
+        ("tensorpipe/tensorpipe_cuda.h", ("tensorpipe/tensorpipe_hip.h", CONV_INCLUDE, API_RUNTIME)),
```

Comment:
@pruthvistony As discussed, we should probably move this to a separate section for pytorch extensions, since more cases like this might come up in the future.

Reply:
Added 'API_PYT_EXT'  to identify these cases.